src/routes/Categories.js

```diff
@@ -1,7 +1,16 @@
 import React from 'react';
+import { useDispatch } from 'react-redux';
+import { checkStatus } from '../redux/categories/categoriesSlice';
 
-const Categories = () => (
-  <button type="button">Check status</button>
-);
+const Categories = () => {
+  const dispatch = useDispatch();
+  const handleCheckStatus = () => {
+    dispatch(checkStatus());
+  };
```

Comment:
- [ ] It would be better if we implement the functionality where a user clicks on the `check status` button, a text `Under construction` is present to the UI. Let us kindly implement this functionality
[Screencast from 03-09-2023 10:40:35 AM.webm](https://user-images.githubusercontent.com/46388113/224000609-9cd23344-74da-4293-97b6-42970056c3bf.webm)